x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/preconfigured_connector_flyout.tsx

```diff
@@ -0,0 +1,131 @@
+  const closeFlyout = useCallback(() => setFlyoutVisibility(false), [setFlyoutVisibility]);
+  if (!flyoutVisible) {
+    return null;
+  }
+
+  const actionTypeModel = actionTypeRegistry.get(initialConnector.actionTypeId);
+
+  return (
+    <EuiFlyout onClose={closeFlyout} aria-labelledby="flyoutActionAddTitle" size="m">
+      <EuiFlyoutHeader hasBorder>
+        <EuiFlexGroup gutterSize="s" alignItems="center">
+          {actionTypeModel ? (
+            <EuiFlexItem grow={false}>
+              <EuiIcon type={actionTypeModel.iconClass} size="m" />
+            </EuiFlexItem>
+          ) : null}
+          <EuiFlexItem>
+            <EuiTitle size="s">
+              <h3 id="flyoutTitle">
+                <FormattedMessage
+                  defaultMessage="{connectorName}"
+                  id="xpack.triggersActionsUI.sections.preconfiguredConnectorForm.flyoutTitle"
+                  values={{ connectorName: initialConnector.name }}
+                />
+                &emsp;
+                <EuiBetaBadge
+                  label="Pre-configured"
+                  tooltipContent={i18n.translate(
+                    'xpack.triggersActionsUI.sections.preconfiguredConnectorForm.tooltipContent',
+                    {
+                      defaultMessage: 'This connector is preconfigured and not allowed to edit',
+                    }
+                  )}
+                />
+                &emsp;
+                <EuiBetaBadge
+                  label="Beta"
+                  tooltipContent={i18n.translate(
+                    'xpack.triggersActionsUI.sections.preconfiguredConnectorForm.betaBadgeTooltipContent',
+                    {
+                      defaultMessage:
+                        '{pluginName} is in beta and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.',
+                      values: {
+                        pluginName: PLUGIN.getI18nName(i18n),
+                      },
+                    }
+                  )}
+                />
+              </h3>
+            </EuiTitle>
+            <EuiText size="s">
+              <FormattedMessage
+                defaultMessage="{actionDescription}"
+                id="xpack.triggersActionsUI.sections.preconfiguredConnectorForm.actionTypeDescription"
+                values={{ actionDescription: actionTypeModel.selectMessage }}
+              />
+            </EuiText>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiFlyoutHeader>
+      <EuiFlyoutBody>
+        <EuiText>
+          {i18n.translate(
+            'xpack.triggersActionsUI.sections.preconfiguredConnectorForm.descriptionText',
+            {
+              defaultMessage: 'This connector is readonly.',
+            }
+          )}
+        </EuiText>
+        <EuiLink href="https://www.elastic.co/guide" target="_blank">
```

Comment:
We'll need documentation for how to pre-configure connectors. I'm thinking this place seems best: https://github.com/elastic/kibana/blob/master/docs/user/alerting/action-types.asciidoc

Once those are published (ex here: https://www.elastic.co/guide/en/kibana/master/action-types.html), we can use the doc links plugin to generate a URL.

I can setup a meeting if you like on how to write asciidocs, the alerting team could benefit as well.

Reply:
Created an issue to cove this